cf_deployment_functions.sh

```diff
@@ -43,7 +43,7 @@ perform_blue_green_deployment() {
   cf map-route ${GREEN_APP} ${CF_PUBLIC_DOMAIN} --hostname ${ROUTE}
 
   echo "# run smoke tests"
-  (${SMOKE_TESTS} ${ROUTE}.${CF_PUBLIC_DOMAIN})
+  (${POST_DEPLOYMENT_TESTS} ${ROUTE}.${CF_PUBLIC_DOMAIN})
```

Comment:
Can we add a test here to see if POST_DEPLOYMENT_TESTS has been set? Then we don't need to set it for the CI builds

Reply:
And an echo statement indicating whether they're being run